### ISSUE TITLE
Rename atomicapp examples - remove extra *atomicpp*, *centos* strings from name

### ISSUE THREAD

**Issue by navidshaikh** [projectatomic/nulecule#163]
@kbsingh suggested about renaming atomicapp examples. Presently the example names contain name of the base image used and "atomicapp" in the name.
The motivation behind renaming is to provide closer user experience as with other tools, for eg:
`yum install appname` and `atomic install appname` or `atomic run appname`

Edit:
Updated the analogy 
`yum install appname` --> `atomic install atomicapp/appname`

**Comment by vpavlin** [comment]
I would agree with this if we could somehow filter/search/pull based on labels - i.e. you can have an alias for something like `alias nulecule="atomic --type nulecule --os centos7" so that you can then do`nulecule run wordpress`which would actually run some image called`wordpress`with labels`type=nulecule`and`os=centos7`.

As we cannot do this, I would suggest to keep the naming as is to distinguish between non-Nulecule images and Nulecule images. 

Same for the OS part of the name.

**Comment by navidshaikh** [comment]
non-Nulecule images themselves may not constitute an app, they might require to link to other (for eg db) containers to make an app. Nulecule examples already have namespace `projectatomic`, so the repository name becomes `projectatomic/$app_name` - I think this should suffice to describe the app in Nulecule-app OR we modify the namespace to be a bit specific as `atomicapp/$app_name` OR `nulecule/$app_name` ?
Regarding other non-Nulecule images - if they are from ISV, they should have ISV namespace like [gitlab/gitlab-ce](https://hub.docker.com/r/gitlab/gitlab-ce/), OR for other images say [fedora/apache](https://hub.docker.com/r/fedora/apache/).

Regarding the OS part - should we really categorize based on base image(s) ? 
I think, for containers, one should not really need to know about the base image, app is layered upon.

**Comment by goern** [comment]
Well, I am super interested in this part, as I dont want to have my
 developers based their work on some random ubuntu image they... esp
 if I am a big huge GOV or Bank ...

**Comment by bexelbie** [comment]
Shouldn't step 1 of using any docker image be to read the Dockerfile?  I can label my image Fedora and use a Centos base.  The point being that I think these labels are a bit of clutter, especially as some examples will deliberately include combinations of stabilized code from a RHEL or CentOS with bleeding edge code from Fedora.  What do we put in the name for those?
